CleanWeather/Models/Weather.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Weather.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 12/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+struct Weather: Codable {
+    
+    enum SFsymbol: String {
+        case clearDay = "clear-day"
+        case clearNight = "clear-night"
+        case cloudy = "cloudy"
+        case fog = "fog"
+        case partlyCloudyDay = "partly-cloudy-day"
+        case partlyCloudyNight = "partly-cloudy-night"
+        case rain = "rain"
+        case sleet = "sleet"
+        case snow = "snow"
+        case wind = "wind"
+        case umbrella = "umbrella"
```

Comment:
jak case masz taki sam jak wartosc to nie musisz robić tego, wystarczy 

case wind
case snow = "snow"
case partlyCloudyDay = "partly-cloudy-day"